spec/ruby/library/bigdecimal/round_spec.rb

```diff
@@ -187,4 +187,10 @@
       @n2_49.round(0, BigDecimal::ROUND_HALF_EVEN).should == @neg_two
     end
   end
+
+  it 'raise exception, if self is special value' do
+    lambda { BigDecimal('NaN').round }.should raise_error(FloatDomainError)
+    lambda { BigDecimal('Infinity').round }.should raise_error(FloatDomainError)
+    lambda { BigDecimal('-Infinity').round }.should raise_error(FloatDomainError)
+  end
```

Comment:
You can commit specs directly to the repository, but they should be in isolated commits.  That way @eregon can cherry-pick them back to the upstream repo.  It gets a lot harder if you've mixed the spec changes in with other changes unrelated to the rubyspec project.